package.json

```diff
@@ -47,17 +47,18 @@
     "@types/jasmine": "^2.8.6",
     "child-process-promise": "^2.2.1",
     "colors": "^1.2.1",
+    "coveo-search-ui": "^2.6459.9",
     "dts-webpack-plugin": "^0.0.9",
     "exports-loader": "^0.7.0",
     "lint-staged": "^7.0.0",
+    "jquery": "^3.2.1",
```

Comment:
What's the reason for using jQuery with this repo ?

Reply:
To simulate jQuery in the page:

https://github.com/coveo/search-ui-tests/blob/master/src/Simulate/JQueryModule.ts